WebRoot/js/zimbraAdmin/search/controller/ZaSearchListController.js

```diff
@@ -366,35 +366,36 @@ function(params) {
 
 	if(isAliasSpec) {
           this.scrollSearchParams={
-                        query:this._currentQuery,//params.query,
-                        types:params.types,
-                        showBusy:true,
-                        busyId:busyId,
-                        busyMsg:ZaMsg.BUSY_SEARCHING,
-                        skipCallbackIfCancelled:false,
-                        sortBy:params.sortBy,
-                        sortAscending:this._currentSortOrder,
-                        attrs:ZaSearch.standardAttributes,
-                        controller:this,
-                        scrollType:"isAliasSearch",
-                        postCallback: postCallback
+            query:this._currentQuery,//params.query,
+            types:params.types,
+            showBusy:true,
+            busyId:busyId,
+            busyMsg:ZaMsg.BUSY_SEARCHING,
+            skipCallbackIfCancelled:false,
+            sortBy:params.sortBy,
+            sortAscending:this._currentSortOrder,
+            attrs:ZaSearch.standardAttributes,
+            controller:this,
+            scrollType:"isAliasSearch",
+            postCallback: postCallback
```

Comment:
indentation to use consistent spaces or tab key.

Reply:
Fixed